nslsii/areadetector/xspress3.py

```diff
@@ -136,11 +136,14 @@ class Xspress3ExternalFileReference(Signal):
 
     """
 
-    def __init__(self, *args, dtype_str="uint32", bin_count=4096, dim_name="bin_count", **kwargs):
+    def __init__(self, *args, dtype_str="uint32", bin_count=4096, dim_name=["frame_number", "number_channels", "bin_count"], **kwargs):
```

Comment:
We should avoid mutable defaults this can be a tuple.

Reply:
Changed to tuple